aryn_playground_cloud_console.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Using the Playground and Console
+description: "Using the Aryn Playground and Aryn Cloud Console"
+---
+
+## Aryn Playground
+
+The quickest way to experiment with the DocPrep and DocParse UIs is through the [Aryn Playground](https://play.aryn.cloud). You can access the Playground with just an ARyn Cloud API key.
```

Comment:
experiment with DocPrep and DocParse is through the Aryn Playground UIs?

Reply:
Ack